postgresql.html

```diff
@@ -56,7 +56,7 @@
 					<i class="fa fa-lock"></i>
 					<span data-i18n="postgresql.label.password"></span>
 				</label>
-				<input type="password" id="node-config-input-password" data-i18n="[placeholder]postgresql.placeholder.password" style="width: 80%;" />
+				<input type="text" id="node-config-input-password" data-i18n="[placeholder]postgresql.placeholder.password" style="width: 80%;" />
```

Comment:
Can't we keep the `password` type and just remove the on change event?

Reply:
I thought about that, but then I thought about when the user selects a password from an environment variable such as `PGPASSWORD` which would then be invisible even if it is not a sensitive string, so that is not ideal either. Possibly the type of the "password" field should be set with radio buttons but I don't know what is best.

Reply:
```suggestion
				<input type="password" id="node-config-input-password" data-i18n="[placeholder]postgresql.placeholder.password" style="width: 80%;" />
```

Ok, then I suggest we use the standard password type for now, and come back to it if anyone reports a problem.